src/main/webapp/search.js

```diff
@@ -110,3 +109,14 @@ function getRequestUrl() {
   url = URL_STRUCTURE + userSearchInput + '&key=' + YOUTUBE_API_KEY;
   return url;
 }
+
+/**
+ * Set apiKey variable to data read from file
+ */
+function retrieveApiKey() {
+  $(function() {
+    $.get('apiKey', function(data) {
```

Comment:
where is this data coming from?

Reply:
I'm not entirely sure but it seems to be the result of $.get('apikey')